scripts/script.yaml

```diff
@@ -60,12 +58,14 @@
             - match: returning                                                     # <--- Previous reports found on device
               steps:
               - say: עבור מי ברצונך לדווח עכשיו?
+              - say: (הכי טוב כשכולם מדווחים בעצמם
```

Comment:
parentheses in a chat is weird
especially when split between bubbles

Reply:
we removed this sentence from the returning script for a reason, why bring it back?

Reply:
It wasn't removed, just appeared at the top - for everyone. I brought it back to prevent it from appearing to the first reporter... do you think it is a mistake?

Reply:
Reminder:  #116 

Read this comment (by @noamoss):

כרגע זה מופיע רק בתחילת הדרך - למשתמשים חדשים.
מכיוון שהתלונה לא חזרה בשבועיים האחרונים, סוגר את הדיון.

Reply:
?